contrib/connector/connector.go

```diff
@@ -21,6 +21,7 @@ import (
 type Connector interface {
 	Attach(map[string]interface{}) (string, error)
 	Detach(map[string]interface{}) error
+	GetInitiatorInfo() (string, error)
```

Comment:
I think we can define a struct `InitiatorInfo` with definition below:
```go
type InitiatorInfo struct {
    HostName string `json:"hostName"`
    InitiatorData map[string]interface{} `json:"initiatorData"`
}
```

Just refer to [CSI Plugin Refactoring](https://docs.google.com/presentation/d/1w40ccIFqN2d3v5T7V3gYYl0RZpqEUbn1WhtmNA93YXE/edit?usp=sharing)